cves/CVE-2012-2779.yml

```diff
@@ -191,8 +207,11 @@ autodiscoverable:
 
     The answer field should be boolean. In answer_note, please explain
     why you come to that conclusion.
-  answer_note:
-  answer:
+  answer_note: | 
+    The vulnerability was that there was no check regarding the validity 
+    of the gop header being decoded. Thus an automated tool could easily discover this 
+    vulnerablity by checking the validity of said gop header.
```

Comment:
Describe that an automated file corrupter could do the trick. There is a chance that it could corrupt the GoP header in just the right way.